src/widgets.js

```diff
@@ -257,16 +257,24 @@ define([
                 var $this = $(this),
                     datavalue = $this.attr('data-value'),
                     match =  datavalue.match('output value="(.*)"'),
-                    value = match ? match[1] : $this.attr('data-value');
-                $this.siblings('.cke_widget_drag_handler_container').children().popout({
-                    title: '',
-                    content: value,
-                    template: '<div contenteditable="false" class="popover">' +
-                        '<div class="arrow"></div>' +
+                    xpath = match ? match[1] : datavalue,
+                    displayId = $this.clone().children().remove().end().text(),
+                    labelMug, labelText;
+                if (/^\/data\//.test(xpath)) {
+                    labelMug = mug.form.getMugByPath(xpath);
+                    labelText = labelMug ? labelMug.p.labelItext.get() : "";
+                } else {
+                    return;
```

Comment:
Discussed with @amsagoff and we decided to disable the popover for case properties until we have support for #case_type/property special syntax.